rudder-agent/SOURCES/rudder-agent-postinst

```diff
@@ -229,6 +229,18 @@ then
   echo " Done."
 fi
 
+# Create a cert if we don't have one yet
+if [ ! -f ${CFE_DIR}/ppkeys/localhost.cert ]
+then
+  if [ -x /opt/rudder/bin/openssl ]
+  then
+    OPENSSL="/opt/rudder/bin/openssl"
+  else
+    OPENSSL="openssl"
+  fi
+  ${OPENSSL} req -new -sha256 -key ${CFE_DIR}/ppkeys/localhost.priv -out ${CFE_DIR}/ppkeys/localhost.cert -passin "pass:Cfengine passphrase" -x509 -days 3650 -subj "/CN=$(hostname)"
```

Comment:
we should have the same profile as for windows agents:
`"$rudderBase\bin\openssl.exe`" req -new -sha256 -key `"$rudderBase\etc\ssl\localhost.priv`" -out `"$rudderBase\etc\ssl\localhost.cert`" -passin `"pass:Rudder-dsc passphrase`" -x509 -days 3650 -extensions agent_cert -subj `"/CN=$env:computername/UID=$guid`""